package.json

```diff
@@ -127,7 +127,7 @@
     "lodash": "^4.17.19",
     "loglevel": "^1.4.1",
     "luxon": "^1.24.1",
-    "metamask-logo": "^2.1.4",
+    "metamask-logo": "danjm/metamask-logo#lookAtAndRender",
```

Comment:
We should PR https://github.com/MetaMask/metamask-logo with this.

Reply:
already in progress: https://github.com/MetaMask/metamask-logo/pull/30